src/schema/nmdc.yaml

```diff
@@ -350,8 +342,6 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:dobj-{id_shoulder}-{id_blade}{id_version}{id_locus}"
           interpolated: true
-    comments:
-      - removed id_prefixes value of GOLD, since we are now using pattern-based id validation
```

Comment:
Was this removed since we don't need it anymore? I'm not sure I understand what this is even saying?